### PR TITLE
Add guide: Monitor and visualize anomalies with Netdata

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Part 2 of the anomaly detection guide started with #10451. This one goes into the alarms/charts created by the collector, along with a stress test to showcase how they act during an incident.

##### Component Name

area/docs

### CODE REVIEW

File: docs/guides/monitor/visualize-monitor-anomalies.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+![An example anomaly probability
+alarm](https://user-images.githubusercontent.com/1153921/104225767-0a0a9480-5404-11eb-9bfd-e29592397203.png)
+
+The second class triggers warning alarms when the number of anomalies in the last two minutes hits 10 or higher.
+
+![An example anomaly count
+alarm](https://user-images.githubusercontent.com/1153921/104225769-0aa32b00-5404-11eb-95f3-7309f9429fe1.png)
+
+If you see either of these alarms in Netdata Cloud, the local Agent dashboard, or on your preferred notification
+platform, it's a safe bet that the nodes current metrics have deviated from normal. That doesn't necessarily mean
```
Comment (aabatangle) [inline docs/guides/monitor/visualize-monitor-anomalies.md:69]: node's current

File: docs/guides/monitor/visualize-monitor-anomalies.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+there's a full-blown incident, depending on what application/service you're using anomaly detection on, but it's worth
+further investigation.
+
+As you use the anomalies collector, you may find that the default settings provide too many or two few genuine alarms.
```
Comment (aabatangle) [inline docs/guides/monitor/visualize-monitor-anomalies.md:73]: too few

File: docs/guides/monitor/visualize-monitor-anomalies.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+preconfigured to visualize per-second anomaly metrics based on your [configuration in
+`anomalies.conf`](/docs/guides/monitor/anomaly-detection.md#configure-the-anomalies-collector).
+
+These charts have the contexts `anomalies.probability` and `anomalies.anomaly`. Together, these charts
+create meaningful visualizations for immediately recognizing not only that something is going wrong on your node, but
+give context as to where to look next.
+
+The `anomalies_local.probability` chart shows the probability that the latest observed data is anomalous, based on the
+trained model. The `anomalies_local.anomaly` chart visualizes 0&rarr;1 predictions based on whether the latest observed
+data is anomalous based on the trained model. Both charts have an equal number of
```
Comment (aabatangle) [inline docs/guides/monitor/visualize-monitor-anomalies.md:92]: incomplete sentence?

File: docs/guides/monitor/visualize-monitor-anomalies.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+These charts have the contexts `anomalies.probability` and `anomalies.anomaly`. Together, these charts
+create meaningful visualizations for immediately recognizing not only that something is going wrong on your node, but
+give context as to where to look next.
+
+The `anomalies_local.probability` chart shows the probability that the latest observed data is anomalous, based on the
+trained model. The `anomalies_local.anomaly` chart visualizes 0&rarr;1 predictions based on whether the latest observed
+data is anomalous based on the trained model. Both charts have an equal number of 
+
+In other words, the `probability` chart shows the amplitude of anomaly, whereas the `anomaly` chart provides quick
```
Comment (aabatangle) [inline docs/guides/monitor/visualize-monitor-anomalies.md:94]: amplitude of the anomaly?

File: docs/guides/monitor/visualize-monitor-anomalies.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+yes/no context.
+
+![Two charts created by the anomalies
+collector](https://user-images.githubusercontent.com/1153921/104226380-ef84eb00-5404-11eb-9faf-9e64c43b95ff.png)
+
+Before `08:32:00`, both charts show little in the way of verified anomalies. Based on the metrics the anomalies
+collector has trained on, a certain percentage of anomaly probably score is normal, as seen in the
```
Comment (aabatangle) [inline docs/guides/monitor/visualize-monitor-anomalies.md:101]: probably --> probability?

File: docs/guides/monitor/visualize-monitor-anomalies.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+`web_log_nginx_requests_prob` dimension and a few others. What you're looking for is large deviations from the "noise"
+in the `anomalies.probability` chart, or any increments to the `anomalies.anomaly` chart.
+
+Unsurprisingly, the stress test ran at `08:32:00` caused significant changes to these charts. The three dimensions that
```
Comment (aabatangle) [inline docs/guides/monitor/visualize-monitor-anomalies.md:105]: that ran?

File: docs/guides/monitor/visualize-monitor-anomalies.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+immediately shot to 100% anomaly probability, and remained there during the test, were `web_log_nginx.requests_prob`,
+`nginx_local.connections_accepted_handled_prob`, and `system.cpu_pressure_prob`. 
+
+## Build an anomaly detection dashboard
+
+[Netdata Cloud](https://app.netdata.cloud) features a drag-and-drop [dashboard
+editor](/docs/visualize/create-dashboards.md) that helps you create entirely new dashboards with charts target for your
```
Comment (aabatangle) [inline docs/guides/monitor/visualize-monitor-anomalies.md:112]: targeted?

File: docs/guides/monitor/visualize-monitor-anomalies.md [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+
+## What's next?
+
+Between this guide and [part 1](/docs/guides/monitor/anomaly-detection.md), which covered setup and configuration, you
+now have a fundamental understanding of how unsupervised anomaly detection in Netdata works, from root cause to alarms
+to preconfigured or custom dashboards.
+
+We'd love to hear your feedback on whether the anomalies collector. Hop over to the [community
+forum](https://community.netdata.cloud/c/agent-development/9) and let us know if you're already getting value from
+unsupervised anomaly detection, or would like to see something added to it. You might even post a custom configuration
```
Comment (aabatangle) [inline docs/guides/monitor/visualize-monitor-anomalies.md:134]: forum, and let

**Review by aabatangle — DISMISSED** [review]
I think we need to add links to parts 1/2 to the Incident Response solution page

**Comment by joelhans** [comment]
@aabatangle Thanks for the edits. I've added another item to the Trello checklist for adding these to the Incident Response page.

File: docs/guides/monitor/visualize-monitor-anomalies.md [hunk 9]
```diff
@@ -0,0 +1,147 @@
+<!--
+title: "Monitor and visualize anomalies with Netdata"
```
Comment (jlbriston) [inline docs/guides/monitor/visualize-monitor-anomalies.md:2]:
```suggestion
title: "Monitor and visualize anomalies with Netdata (Part 2)"
```

File: docs/guides/monitor/visualize-monitor-anomalies.md [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+investigation using [Metric Correlations](https://learn.netdata.cloud/docs/cloud/insights/metric-correlations) to narrow
+the dashboard into only the charts relevant to what you're seeing from the anomalies collector.
+
+## What's next?
+
+Between this guide and [part 1](/docs/guides/monitor/anomaly-detection.md), which covered setup and configuration, you
+now have a fundamental understanding of how unsupervised anomaly detection in Netdata works, from root cause to alarms
+to preconfigured or custom dashboards.
+
+We'd love to hear your feedback on whether the anomalies collector. Hop over to the [community
```
Comment (jlbriston) [inline docs/guides/monitor/visualize-monitor-anomalies.md:133]:
```suggestion
We'd love to hear your feedback on the anomalies collector. Hop over to the [community
```

**Review by jlbriston — DISMISSED** [review]

**Review by aabatangle — APPROVED** [review]

**Review by jlbriston — APPROVED** [review]

**Pull request merged**